src/wrappers/EEPROM_TC.cpp

```diff
@@ -31,11 +33,19 @@ float EEPROM_TC::eepromReadFloat(uint16_t address) {
   return value;
 }
 
+/**
+ * @brief writes the value to the address and triggers an alert in DataLogger
+ *
+ * @param address
+ * @param value
+ */
 void EEPROM_TC::eepromWriteFloat(uint16_t address, float value) {
-  assert(address >= 0);
```

Comment:
Did this assertion do anything?

Reply:
Not for an unsigned number! I'll change it to `!= 0`.

Reply:
So the typecasting made the ">= 0" assertion meaningless, I guess?
Regarding "!= 0" though, isn't PH_ADDRESS zero? We want to allow setPH() and getPh() to work, right? So remove these assertions entirely?

Reply:
You are right, they should be removed entirely. If we had it to do over, it might have been wise to reserve address zero as unused (location zero in memory is a real location but it is often left unused and treated as an error to catch null pointers when designing operating systems).